src/pipeline_v3/cli_main.py

```
@@ -13,33 +13,17 @@
 import asyncio
 import logging
 import sys
-from pathlib import Path
 
-# Add the current directory to Python path for both relative and absolute imports
-sys.path.insert(0, str(Path(__file__).parent))
-sys.path.insert(0, str(Path(__file__).parent.parent.parent))
+# Import using relative imports
+from cli.management import main
```

Pylint (3.3.7)

[error] Unable to import 'cli.management'

(E0401)